plugins/inputs/opcua/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Input Plugin: opcua_client
+
+The opcua_client plugin retrieves data from OPCUA slave devices
+
+### Configuration:
+
+```toml
+
+# ## Connection Configuration
+#  ##
+#  ## The plugin supports connections to PLCs via OPCUA
+#  ##
+#  ## Device name
+name = "opcua_rocks"
+#
+#  # OPC UA Endpoint URL
+endpoint = "opc.tcp://opcua.rocks:4840"
+#
+#  ## Read Timeout
+#  ## add an arbitrary timeout (seconds) to demonstrate how to stop a subscription
+#  ## with a context.
+timeout = 30
+#
+#  # Time Inteval, default = 10s
+time_interval = "5s"
+#
+#  # Security policy: None, Basic128Rsa15, Basic256, Basic256Sha256. Default: auto
+security_policy = "None"
+#
+#  # Security mode: None, Sign, SignAndEncrypt. Default: auto
+security_mode = "None"
+#
```

Comment:
Include auto in the list of valid values.  Don't set them to "None" in the sample config if they default to auto.

Also, where in the code do these default to auto?  I didn't find it in a quick review.